src/main/java/org/zalando/planb/provider/CassandraConfig.java

```diff
@@ -19,6 +20,7 @@ Session initializeCassandra() {
         final Cluster.Builder builder = Cluster.builder();
 
         builder.addContactPoints(cassandra.getContactPoints().split(","));
+        builder.withAddressTranslator(new EC2MultiRegionAddressTranslator());
```

Comment:
Then let's try with Spring profiles and some conditional beans. E.g. Could
be enabled with environment variable SPRING_PROFILES_ACTIVE: aws in our
senza files.
Am 17.03.2016 10:49 nachm. schrieb "Luis" <notifications@github.com>:

> In src/main/java/org/zalando/planb/provider/CassandraConfig.java
> <https://github.com/zalando/planb-provider/pull/73#discussion_r56583634>:
>
> > @@ -19,6 +20,7 @@ Session initializeCassandra() {
> >          final Cluster.Builder builder = Cluster.builder();
> >
> >          builder.addContactPoints(cassandra.getContactPoints().split(","));
> > +        builder.withAddressTranslator(new EC2MultiRegionAddressTranslator());
>
> You got me at GCE :P
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/planb-provider/pull/73/files/7d479a27453b1bd8945be9e28ab19b6a2ae4c8a5#r56583634>
>